modules/33acrossAnalyticsAdapter.js

```diff
@@ -474,8 +568,8 @@ function getLogger() {
   const LPREFIX = `${PROVIDER_NAME} Analytics: `;
 
   return {
-    info: (msg, ...args) => logInfo(`${LPREFIX}${msg}`, ...args),
-    warn: (msg, ...args) => logWarn(`${LPREFIX}${msg}`, ...args),
-    error: (msg, ...args) => logError(`${LPREFIX}${msg}`, ...args),
+    info: (msg, ...args) => logInfo(`${LPREFIX}${msg}`, ...JSON.parse(JSON.stringify(args))),
```

Comment:
is the intention of JSON.parse + JSON.stringify to filter out some undefined values?

I think it would be good to have a function defined for this purpose and avoid repetition.

Reply:
That's a good thought, but I actually did this to ensure that in the browser logs, you see the data *as it was* rather than the data *as it is now*. It's a deepCopy, but we don't need a true clone that would preserve functions, just to write out the data. It's also a bit faster than structuredClone. But I just benchmarked that against the prebid `clone` function, and that's much faster still, so I'll switch to that for this purpose.